File: app-legacy-migration-test/src/legacy/java/apptentive/com/android/feedback/test/MyApplication.java
package apptentive.com.android.feedback.test;


import androidx.multidex.MultiDexApplication;

import com.apptentive.android.sdk.Apptentive;
import com.apptentive.android.sdk.ApptentiveConfiguration;
import com.apptentive.android.sdk.ApptentiveLog;

public class MyApplication extends MultiDexApplication {
    @Override
    public void onCreate() {
        super.onCreate();

        ApptentiveConfiguration configuration = new ApptentiveConfiguration("ANDROID-ANDROID-DEV-c9c0b324114f", "98f5539e9310dc290394c68b76664e98");
        configuration.setLogLevel(ApptentiveLog.Level.VERBOSE);
        configuration.setShouldEncryptStorage(true);
        Apptentive.register(this, configuration);
    }
}

File: app-ui-test/src/main/java/apptentive/com/app/test/MyApplication.kt
package apptentive.com.app.test

import androidx.multidex.MultiDexApplication
import apptentive.com.android.core.AndroidLoggerProvider
import apptentive.com.android.core.DependencyProvider

class MyApplication:  MultiDexApplication() {
    override fun onCreate() {
        super.onCreate()

        // register dependency providers
        DependencyProvider.register(AndroidLoggerProvider("Apptentive"))
        // DependencyProvider.register(AndroidExecutorFactoryProvider())
        // DependencyProvider.register(AndroidFileSystemProvider(applicationContext, "apptentive.com.android.feedback"))
    }
}